src/components/patientPage/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Header from "components/header";
+import PatientCard from "components/patientPage/patientCard";
+import HospitSummary from "components/patientPage/hospitSummary";
+import TimelinePatient from "components/patientPage/timelinePatient";
+import { Patient } from "types";
+import { getPatientData } from "services/api";
+
+import "./style.css";
+
+interface Props {
+  patientId: string;
+}
+
+const PatientPage = ({ patientId }: Props) => {
+  const [patientData, setPatientData] = React.useState({} as Patient);
+
+  React.useEffect(() => {
+    const fetchPatientData = async () => {
+      const patient: any = await getPatientData(patientId);
+      setPatientData(patient);
+    };
+    fetchPatientData();
+  }, [patientId]);
+
+  return (
+    <>
+      <Header patient={patientData} />
+      <div className="patientFill">
+        <div className="patientGeneral">
+          <div className="patientCard bp3-card bp3-elevation-3 ">
```

Comment:
can you import `Card` from blueprint.js and use `<Card classsName="patientCard"></Card>` instead of passing the className ? or are you trying to do something different ?

Reply:
Converted to Card components